### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/laravel-doctrine/orm/pull/517/commits/4c23cddd950d3ba4fae6d07a5650efe8817fb971?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/orm.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "laravel-doctrine/orm": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        php: ['7.3', '7.4', '8.0', '8.1']
+        php: [8.0]
```
Comment (jasonmccreary) [inline .github/workflows/ci.yml:10]: Feel free to re-add `8.1`.

File: composer.json [hunk 2]
```diff
@@ -16,32 +16,32 @@
...
+        "illuminate/auth": "^9.0",
+        "illuminate/console": "^9.0",
+        "illuminate/container": "^9.0",
+        "illuminate/contracts": "^9.0",
+        "illuminate/pagination": "^9.0",
+        "illuminate/routing": "^9.0",
+        "illuminate/support": "^9.0",
+        "illuminate/validation": "^9.0",
+        "illuminate/view": "^9.0",
         "symfony/serializer": "^2.7|^3.0|^4.0|^5.0",
```
Comment (jasonmccreary) [inline composer.json:34]: This should probably be bumped to `^6.0` if you plan on only supporting Laravel 9.

**Comment by eigan** [comment]
There were some issues here which was handled better in #518 (we do not support dbal:3, cache:2, and better matrix). Thanks anyway. Cool tool 🎉  🙂

**Comment by jasonmccreary** [comment]
@eigan no problem. We'll try to make some tweaks for next time. 👍

**Pull request closed without merging**